llvm/lib/Object/ELF.cpp

```diff
@@ -392,6 +393,70 @@ ELFFile<ELFT>::decode_relrs(Elf_Relr_Range relrs) const {
   return Relocs;
 }
 
+template <class ELFT>
+uint64_t ELFFile<ELFT>::crelHeader(ArrayRef<uint8_t> Content) const {
+  DataExtractor Data(Content, true, 8); // endian/class is irrelevant
```

Comment:
Endian/class may be irrelevant, but since you have access to the ELFT here anyway, would it not make sense to just use the corresponding values? Same elsewhere.